### DISCUSSION TITLE
How does WXT compare to using Vite with @crxjs/vite-plugin ?

### DISCUSSION THREAD

**Discussion by lucasrmendonca** [wxt-dev/wxt#496] [Q&A] [answered]
I currently have an extension using Webpack and stumbled upon this repository while investigating how to migrate it from Webpack to Vite

**Comment by aklinker1** [comment, marked as answer]
See https://wxt.dev/guide/compare.html and https://github.com/wxt-dev/wxt/discussions/493. If you have any more questions, I'll answer them!
